### PR TITLE
build: missing repo fields for publish

### PR DESCRIPTION
**Pull request by peterpeterparker**
# Motivation

Repo fields are missing in `package.json` and therefore we cannot publish to npm.

See https://github.com/dfinity/eslint-config-oisy-wallet/actions/runs/11053281386/job/30707280793

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,6 +3,13 @@
   "version": "0.0.1",
   "license": "Apache-2.0",
   "description": "Shared ESLint configurations from the Oisy Wallet team",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/dfinity/eslint-config-oisy-wallet.git"
+  },
+  "bugs": {
+    "url": "https://github.com/dfinity/eslint-config-oisy-wallet"
```
Comment (AntonioVentilii-DFINITY) [inline package.json:11]: shall this point directly to `/issues`

**Review by AntonioVentilii-DFINITY — APPROVED** [review]
LGTM, just a small question

File: package.json (hunk 1 above)
Comment (peterpeterparker) [reply package.json:11]: I actually never do so, don't know why. That's a good point. We can.

Comment (peterpeterparker) [reply package.json]: Done.

**Pull request merged**